actix/src/contextimpl.rs

```diff
@@ -278,7 +278,7 @@ where
 
     /// Restart context. Cleanup all futures, except address queue.
     #[inline]
-    pub(crate) fn restart(&mut self) -> bool
+    pub fn restart(&mut self) -> bool
```

Comment:
if we're making this public, i'd like to see a test added somewhere and also some improved documentation on this method